docker-compose-starburst.yml

```diff
@@ -16,6 +16,37 @@ services:
       POSTGRES_USER: dbt-trino
       POSTGRES_PASSWORD: dbt-trino
 
+  hive-metastore:
+    hostname: hive-metastore
+    image: 'michieldesmet/hive-metastore:latest'
```

Comment:
Let's maybe push it to other more official place?

Reply:
@hovaesco can we maybe get a hms image maintained by Starburst ?

Reply:
Please note that the image is multi arch: it works well on both ARM as AMD based architectures. See https://github.com/innoverio/hive-metastore-multiarch

Reply:
I've raised this topic internally. Ideally, we should have @mdesmet [hive-metastore-multiarch](https://github.com/innoverio/hive-metastore-multiarch) image to use across different Trino projects and integrations for testing. 

HMS image is also used in [trino-getting-started](https://github.com/bitsondatadev/trino-getting-started/blob/main/hive/trino-minio/docker-compose.yml#L27). 

cc @bitsondatadev

Reply:
@mdesmet, it seems like mysql is running in the same container as the metastore, which is probably better, but for training purposes (possibly for testing purposes) we may still want to expose the database `3306` port.

I assume we're never using these in production?

Here's an example where I dive into the database backing HMS:
https://github.com/bitsondatadev/trino-getting-started/tree/main/hive/trino-minio

Reply:
@bitsondatadev: For training purposes it might be better to have mysql and hms separated to clearly demonstrate the components of the architecture. For testing purposes I think bundling them makes sense, similar what we already do in the the hive images in https://github.com/trinodb/docker-images.

Reply:
Yeah, I certainly like the simplicity of an internal db. Perhaps we just need an image that can both, handle multi-arch and by default has an internal mysql with the option to override the settings and point to an external database.